.github/workflows/build-and-test.yaml

```diff
@@ -13,7 +13,7 @@ on:
     - cron: '0 19 * * 1-5'
 
 env:
-  DEFAULT_GO_VERSION: ^1.16
+  DEFAULT_GO_VERSION: 1.16
```

Comment:
It seems like `^1.16` [meant](https://github.com/npm/node-semver#caret-ranges-123-025-004) "any minor version >= 1.16 but < 2.0" (thus, `1.17` or even `1.19` would be acceptable). Removing the caret means we want exactly `1.16`.

Do I understand your intent correctly?